recipes/audiofile/all/test_package/conanfile.py

```diff
@@ -0,0 +1,18 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        if not tools.cross_building(self.settings):
+            bin_path = os.path.join("bin", "test_package")
+            wav_path = os.path.join(self.source_folder, "8kadpcm.wav")
```

Comment:
This file is missing a license.
Also, it might be interesting where you got it from, to know the creator.
When in doubt, I would just [generate a sine using `ffmpeg`](https://superuser.com/questions/724391/how-to-generate-a-sine-wave-with-ffmpeg).

Reply:
It comes from vorbis test_package's recipe.

Reply:
The file  probably comes from here: https://www.nch.com.au/acm/
The site does not mention a license, so I would classify it as non-free.
Albeit a small one, I think this is a license infringement.

Reply:
we may include "torvalds-says-linux.wav" which is part of Linux kernel and shared GNU GPL.
or we can just encode simple .wav file with silence and put there with MIT.
or, we can just make some random bytes and name em - as .wav - test package `audioFile.load` will probably fail, but we don't need it to pass, we only need to ensure it compiles and links correctly.

Reply:
I've replaced the file by a smaller one generated locally

Reply:
Oh just a sine. :disappointed: 
I expected some karaoke from you :joy: :rofl: